profiling/simple-kernel-timer/kp_json_writer.cpp

```diff
@@ -79,18 +79,8 @@ int main(int argc, char* argv[]) {
 		exit(-1);
 	}
 
-        char delimiter   = ' ';
-        int fixed_width  = 0;
-
         int commandline_args = 1;
         while( (commandline_args<argc ) && (argv[commandline_args][0]=='-') ) {
-          if(strcmp(argv[commandline_args],"--delimiter")==0) {
-            delimiter=argv[++commandline_args][0];
-          }
-          if(strcmp(argv[commandline_args],"--fixed-width")==0) {
-            fixed_width=atoi(argv[++commandline_args]);
-          }
-
```

Comment:
These command line arguments were parsed but never used. I'm not quite sure if we want to keep them or not.

Reply:
No that is a leftover from starting from kp_reader.  I think I wrote that guy and the delimiter of fixed width does not make sense for JSON